src/main/java/duke/Storage.java

```diff
@@ -7,6 +7,8 @@
 import java.io.FileReader;
 import java.io.IOException;
 
+import java.util.ArrayList;
+
 /**
  * Represents a storage handler for managing reading from and writing to a file.
```

Comment:
Was kinda confused by phrasing, instead of "this class" i think you meant "this method"?